mobile/Features/Feed/Screens/Customization.jsx

```diff
@@ -0,0 +1,122 @@
+import {
+    Text, View, TouchableOpacity, StyleSheet,
+  } from 'react-native';
+import PropTypes from 'prop-types';
+import Icon from 'react-native-vector-icons/FontAwesome';
+
+const styles = StyleSheet.create({
+    container: {
+        flex: 1,
+        alignItems: 'center',
+        justifyContent: 'center',
+        backgroundColor: '#E3F5F4',
+    },
+    arrowContainer: {
+        position: 'absolute',
+        top: 100,
+        left: 30,
+        flexDirection: 'row',
+        alignItems: 'center',
+    },
+    title: {
+        fontSize: 40,
```

Comment:
Could you put the styling in a separate file? example to reference is JournalStyle.jsx